components/http_foundation/introduction.rst

```diff
@@ -255,8 +255,8 @@ by using the following methods:
 :method:`Symfony\\Component\\HttpFoundation\\Request::getEncodings`
     Returns the list of accepted encodings ordered by descending quality.
 
-  .. versionadded:: 2.4
-      The ``getEncodings()`` method was introduced in Symfony 2.4.
+.. versionadded:: 2.4
+    The ``getEncodings()`` method was introduced in Symfony 2.4.
```

Comment:
According to platform.sh, this change fixes the issue: http://pr-5822-6qmocelev2lwe.eu.platform.sh/components/http_foundation/introduction.html#accessing-accept-headers-data